crypto/dso/dso_dl.c

```diff
@@ -236,23 +236,27 @@ static char *dl_merger(DSO *dso, const char *filespec1, const char *filespec2)
      * if the second file specification is missing.
      */
     if (!filespec2 || filespec1[0] == '/') {
-        merged = OPENSSL_malloc(strlen(filespec1) + 1);
+        size_t size;
+        size = strlen(filespec1) + 1;
```

Comment:
combine these two lines and put a blank line after the local variable.  this occurs a copule of times.